src/main/docker/scripts/export-file-based-secrets.sh

```diff
@@ -0,0 +1,63 @@
+#!/bin/bash
+#
+# Copyright 2017-2021 Micro Focus or one of its affiliates.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+# A function for logging in the caf logging format.
+log() {
+    echo "$@" |& $(dirname "$0")/../scripts/caf-log-format.sh "export-file-based-secrets.sh"
+}
+
```

Comment:
Just directly use `echo` instead.  When calling the script pipe it through the `caf-log-format.sh` script and it will handle putting it into the right format.

Reply:
I couldn't find a way to make that work, for example, this works:

```
source $(dirname "$0")/../scripts/export-file-based-secrets.sh
```

But piping this script to `caf-log-format.sh` doesn't work - I believe because when you pipe to a command, a subshell is created and any variables set inside of that subshell will go away when it exits:

```
source $(dirname "$0")/../scripts/export-file-based-secrets.sh |& $(dirname "$0")/../scripts/caf-log-format.sh "export-file-based-secrets.sh"
```